# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../afpl-ghostpcl/install.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2009-02-24 15:46:47.000000000 +0200
+++ ./Makefile	2009-02-24 15:47:48.000000000 +0200
@@ -1,3 +1,7 @@
+prefix=/usr/local
+bindir=$(prefix)/bin
+DESTDIR=
+
 product:
 	make -C main -f pcl6_gcc.mak	# build PCL and PCLXL. 
 
@@ -13,7 +17,7 @@
 	make -C main -f pcl6_gcc.mak pg-fp
 
 install:
-	install main/obj/pcl6 /usr/local/bin
+	install main/obj/pcl6 $(DESTDIR)$(bindir)/
 
 test: 
 	cd tools; ../main/obj/pcl6 -dTextAlphaBits=4 owl.pcl tiger.px3 # test with PCL and PXL test file 
